src/lib/ai/mental-llama/utils/clinical-relevance.ts

```
@@ -0,0 +1,564 @@
+ * This module implements specialized metrics for evaluating the clinical relevance
+ * of mental health explanations, based on domain-specific criteria including:
+ *
+ * 1. Evidence-based terminology and references
+ * 2. Diagnostic criteria alignment
+ * 3. Treatment relevance
+ * 4. Clinical accuracy
+ * 5. Alignment with established mental health frameworks
+ */
+
+import { logger } from '../../../logging';
+import { MentalLLaMAModelProvider } from '../MentalLLaMAModelProvider';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)